### PR TITLE
Feature/nijo 332 remove booking details

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: HappyTravel.Edo.Data/Booking/LocationInfo.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        {
+            CountryCode = countryCode;
+            CountryName = countryName;
+            CityCode = cityCode;
+            CityName = cityName;
+        }
+        
+        public string CountryCode { get; }
+        public string CountryName { get; }
+        public string CityCode { get; }
```
Comment (kirillta) [inline HappyTravel.Edo.Data/Booking/LocationInfo.cs:15]: It's a locality code. And do we need codes here?

Comment (kirillta) [reply HappyTravel.Edo.Data/Booking/LocationInfo.cs:15]: Also, where's a zone name?

File: HappyTravel.Edo.Data/EdoContext.cs [hunk 2]
```diff
@@ -527,6 +524,26 @@ private void BuildBooking(ModelBuilder builder)
                 booking.Property(b => b.LanguageCode)
                     .IsRequired()
                     .HasDefaultValue("en");
+                
+                booking.Property(b => b.AccommodationId)
+                    .IsRequired();
+                
+                booking.Property(b => b.AccommodationName)
```
Comment (kirillta) [inline HappyTravel.Edo.Data/EdoContext.cs:531]: Do we need the name as a separate column?

Comment (mingazhev) [reply HappyTravel.Edo.Data/EdoContext.cs:531]: Thought that if we have an id as separate column, why not to do the same with name?

File: HappyTravel.Edo.Data/Booking/LocationInfo.cs (hunk 1 above)
Comment (mingazhev) [reply HappyTravel.Edo.Data/Booking/LocationInfo.cs]: Removed codes, renamed accordingly

**Pull request merged**